backend/python/app/services/interfaces/concern_service.py

```diff
@@ -27,22 +27,12 @@ def get_child_concern(self, child_concern):
         """
         pass
 
-    def add_familial_concern(self, familial_concern):
+    def add_concern(self, type, concern):
         """Creates a new db object and returns the ConcernDTO if adding was successful
 
-        :param familial_concern: string of familial_concern
+        :param concern: string of concern, type: type of concern
         :return: ConcernDTO
         :rtype: ConcernDTO
-        :raises Exception: if familial_concern is invalid
-        """
-        pass
-
-    def add_child_concern(self, child_concern):
-        """Creates a new db object and returns the ConcernDTO if adding was successful
-
-        :param child_concern: string of child_concern
-        :return: ConcernDTO
-        :rtype: ConcernDTO
-        :raises Exception: if child_concern is invalid
+        :raises Exception: if concern is invalid
```

Comment:
```suggestion
        :raises Exception: if concern is invalid/adding to database fails
```
are we actually doing any validation on the input in the add method right now?

Reply:
for now I dont think so. I did not see any validation method inside `ConcernDTO` class.